starlark/support.go

```diff
@@ -103,6 +107,7 @@ var (
 		connTimeout int // seconds
 	}{
 		crashdir: filepath.Join(os.Getenv("HOME"), ".crashd"),
+		logpath:  filepath.Join(os.Getenv("HOME"), ".crashd", "crashd.log"),
```

Comment:
Maybe just no default here at all? Or this is fine at the start but then gets swapped out for the UUID specific one? Or UUID generated first and put here as the default for others to pull from?

Reply:
Again, if UUID is to avoid log naming crash, one quick way to resolve is to allow user to specify log file path at the cli.